packages/maigret/PKGBUILD

```diff
@@ -36,9 +36,9 @@ package() {
 #!/bin/sh
 cd /usr/share/$pkgname
 source venv/bin/activate
-exec python $pkgname/$pkgname.py "\$@"
+exec python maigret/maigret.py "\$@"
 EOF
 
-  chmod a+x "$pkgdir/usr/bin/$pkgname"
+chmod a+x "$pkgdir/usr/bin/$pkgname"
 }
```

Comment:
Remove spaces in this closing bracket.

Reply:
Not sure. Here. My editor may have done this ? Lmk if it's ok now. Odd.